lib/src/domain/location/location.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:meta/meta.dart';
+
+class Location {
+  final double latitude;
+  final double longitude;
+
+  const Location({
+    @required this.latitude,
+    @required this.longitude,
+  })  : assert(latitude != null && latitude >= -90 && latitude <= 90,
+            'Latitude must be in an interval from -90.0 to +90.0'),
```

Comment:
mamy klasę 'untranslatable_strings' na takie rzeczy

Reply:
niezupełnie, to nie jest do prezentowania dla użytkownika tylko raczej dla developera, żeby później miał ew. w logach ocb

Reply:
niezupełnie, celem tego nie jest prezentacja użytkownikowi na ui, tylko sygnalizacja w logach ocb w razie wyjątku

Reply:
zakładałem, że wszystkie stringsy tam będą, nie tylko ui, ale rzeczywiście ten plik jest obecnie w folderze UI i w sumie chyba nie ma co niepotrzebnie komplikować